src/main/java/edu/project2/tripplanner/service/CommentServiceImpl.java

```diff
@@ -0,0 +1,88 @@
+package edu.project2.tripplanner.service;
+
+import edu.project2.tripplanner.dto.CommentDTO;
+import edu.project2.tripplanner.dto.CommentIdDTO;
+import edu.project2.tripplanner.exception.Message;
+import edu.project2.tripplanner.exception.NotFoundException;
+import edu.project2.tripplanner.model.Comment;
+import edu.project2.tripplanner.repository.CommentRepository;
+import edu.project2.tripplanner.repository.PlaceRepository;
+import edu.project2.tripplanner.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class CommentServiceImpl implements CommentService, Message {
+    private final CommentRepository commentRepository;
+    private final  UserRepository userRepository;
+    private final PlaceRepository placeRepository;
+
+    private Long userId;
+    private Long placeId;
+
+    @Override
+    public List<Comment> findByUserId(Long userId) {
+
+        return commentRepository.findByUserId(userId);
+    }
+
+    @Override
+    public Optional<Comment> findByIdAndUserId(Long id, Long userId) {
+
+        return commentRepository.findByIdAndUserId(id, userId);
+    }
+
+    @Override
+    public Optional<Comment> findByIdAndUserIdAndPlaceId(Long id, Long userId, Long placeId) {
+
+        return commentRepository.findByIdAndUserIdAndPlaceId(id, userId, placeId);
+    }
+
+    @Override
+    public void deleteCommentById(CommentIdDTO commentIdDTO) {
+         userId=commentIdDTO.getUserId();
```

Comment:
there is no reason to make userId or else - a global variable.
Use just local variables